test/collection_test.js

```diff
@@ -827,9 +827,7 @@ describe("Collection", () => {
       it("should resolve with imported deletions", () => {
         return articles.pullChanges(result)
           .then(res => res.deleted)
-          .should.eventually.become([
-            {id: id_4}
-          ]);
+          .should.eventually.become([id_4]);
```

Comment:
Is this a breaking API change because we can no longer access the old record? Can't the API stay the same?